tools/automator/utils.sh

```diff
@@ -42,3 +42,8 @@ split_on_commas() {
 evaluate_tmpl() {
   bash -c "eval echo \"$1\""
 }
+
+hash() {
+  local val="$1" chars="${2:-8}"
```

Comment:
$chars is always 8, please leave this as a constant until it is used.

Reply:
Yea, this is acting as a function _default_ value. But I get your point that until there is a usage of this function both with and without a need for the default there is not explicit benefit.

Reply:
Converted to a constant.

Reply:
No need for this variable, it is only used once.

Reply:
This is a utility script (`utils.sh`), so as more jobs use automator, we will benefit from a generic function definition with defaults. But I will hardcode until that need arises.

Reply:
Updated